### ISSUE TITLE
Improving python client docs

### ISSUE THREAD

**Issue by riibeirogabriel** [learningOrchestra/pythonClient#2]
@navendu-pottekkat do you can think a way to improve the quality of the docs?

**Comment by riibeirogabriel** [comment]
I did have think a way, but is too much classes and methods in this package.

**Comment by pottekkat** [comment]
I will check it out. We can find some way. I will work on it through the week.

**Comment by riibeirogabriel** [comment]
thanks! I also will think a way.

**Comment by pottekkat** [comment]
@riibeirogabriel Could you possibly comment the example code? And is there any way to split the preprocessing_code other than writing it as preprocessing_code = ```from ...

We can also change the '_' in the imports to '-'

**Comment by riibeirogabriel** [comment]
The preprocessing_code is sended as string in API, and the model builder microservice interpret this string as python 3 code, this code is the right way to work, but maybe we can write a session with the preprocessing_code uncommented, to highlight the python sintax and make the comments, what do you think? and ok, we can change from "_" to "-".

**Comment by pottekkat** [comment]
> The preprocessing_code is sended as string in API, and the model builder microservice interpret this string as python 3 code, this code is the right way to work, but maybe we can write a session with the preprocessing_code uncommented, to highlight the python sintax and make the comments, what you think? and ok, we can change from "_" to "-".

Oh I did not know it worked that way. Then you should do what you think the best. Then you can comment and change the underscores.

I will take a look at the docs repo and the landing page tomorrow. You can also work on getting a domain address if you are interested. It is late here and that is one of the problems when we are working from 2 opposite ends of the world!

**Comment by riibeirogabriel** [comment]
Okay, it will see this and make something, thanks!

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat We can create another repo to serve the landing page in a github.io domain? It is a temp solution, till i provide a domain in next releases of learingOrchestra,and  i don't have knowledge in front end to build a landing page.

**Comment by pottekkat** [comment]
@riibeirogabriel Yes, that is what I was planning to do. We will create a landing page in a new repo. And later we can add the domain. And we can keep the docs in a subdomain like docs.website.com

It would be best if you get the domain ASAP and we can start the promotion. Lot of folks will contribute for hacktoberfest

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat alright, I will provide the domain in this week!

**Comment by riibeirogabriel** [comment]
I have installed the learning-orhcestra-client and success! but to run the python interpreter on .py file, is throwed a exception in "import learning-orchestra-client" as invalid sintax, the PEP8 recommends use underscores in variables, maybe it can be relationed.
![image](https://user-images.githubusercontent.com/33736256/95010971-9b978880-0603-11eb-8c5e-4d206186010a.png)

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat and thanks for all help for this project, you did make the learningOrchestra more professional and friendly to users!

**Comment by pottekkat** [comment]
Then maybe we should change that in the README

**Comment by pottekkat** [comment]
Actually in the readme it is learning_orchestra_client

**Comment by riibeirogabriel** [comment]
Okay

**Comment by pottekkat** [comment]
> Okay

I will make a PR by EOD with the landing page website

**Comment by riibeirogabriel** [comment]
@navendu-pottekkat Maybe we can rename this repo from learningOrchestra-python-client to python-client, what do you think?

**Comment by pottekkat** [comment]
both are fine in my opinion

>
